court_scraper/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We should delete this VSCode configuration file from version control and then update gitignore to ensure it doesn't creep back in the future.